src/web/extractors.rs

```diff
@@ -445,7 +445,12 @@ impl BsoParam {
         // path: "/1.5/{uid}/storage/{collection}/{bso}"
         let elements: Vec<&str> = uri.path().split('/').collect();
         let elem = elements.get(3);
-        if elem.is_none() || elem != Some(&"storage") || elements.len() != 6 {
+        if elem.is_none() {
```

Comment:
Was this for debugging?

Reply:
This is because some of the tests expect to be able to do a DELETE on a weirdly spelled url to reset state. I'd really rather understand how handle this whole chunk better, but this works for now.

Reply:
Ok, I was able to go back to the way the code was originally here.